services/gps2tsdb/requirements.txt

```diff
@@ -0,0 +1,4 @@
+postgres
+pycairo
```

Comment:
isn't pycairo a graphical environment library?

Reply:
Not sure but it does not seem vital. Will be removed with multistage build

Reply:
Not sure, but I believe it is not needed. Should be removed as part of the transition to multistage

Reply:
Not sure, but I don't think it is needed. It should be removed as part of the transition to multistage that I am working on

Reply:
I'm not sure that it will be easy to remove? I assume you will just copy the entire virtualenv directory into the final stage.

Reply:
You are correct... I removed `pycario` from the requirements list but is still seems gets built and copied over... Could be a dependency of `PyGObject`?

Reply:
Likely. DBUS started as a communication tool for use in graphical environments (which is why gdlib wrote a lib for dbus). SystemD made it popular outside of that with sd-bus. Maybe for the gdlib stuff to work it just has to bring in a bunch of the gdlib's other dependencies. In the long run run, we might want to look at Jeepney, a pure Python DBUS client (https://gitlab.com/takluyver/jeepney).  A bit more verbose API (but also less surprising).

Reply:
Agreed. I think we stick with it for now since it works and make an issue to transition to jeepney